config/resource_suppressions.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+    "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+    "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+    
+      <!-- Suppressions inside test and it folders -->
+      <suppress checks="NewlineAtEndOfFile" files="(src[\\/]it[\\/].*rule731selfexplanatory[\\/]InputJavadocMethodCheck\.java|src[\\/]test[\\/].*Newline.*AtEndOfFile\.java|src[\\/]test[\\/].*astprinter[\\/]InputJavadocComment\.javadoc|src[\\/]test[\\/].*astprinter[\\/]InputJavadocWithError\.javadoc|src[\\/]test[\\/].*checks[\\/]InputEmptyFile\.txt|src[\\/]test[\\/].*grammars[\\/]javadoc[\\/]htmlTags[\\/].*|src[\\/]test[\\/].*grammars[\\/]javadoc[\\/]javadocTags[\\/].*|src[\\/]test[\\/].*grammars[\\/]InputRegressionEmptyAst\.txt|src[\\/]test[\\/].*grammars[\\/]javadoc[\\/]InputLeadingAsterisks\.txt|src[\\/]test[\\/].*grammars[\\/]comments[\\/]InputFullOfSinglelineComments.java|src[\\/]test[\\/].*grammars[\\/]InputEmbeddedNullChar\.java)"/>
```

Comment:
Should I add reasons for all suppressions or just a general message at the top saying smth like this "UTs that use these files cannot work with a new line at the end". Imho first approach may be a good practice for future suppressions?